lib/config.js

```diff
@@ -30,6 +30,13 @@ const Config = {
       type: "boolean",
       default: true
     },
+    outputAreaFontSize: {
+      title: "Change fontsize of Output area",
+      includeTitle: false,
+      description: "Change the fontsize of the Output area.",
+      type: "integer",
+      default: "inherit"
```

Comment:
I don't think with integer we can use `"inherit"` as a default value here since it is a string. We'll need to handle this in JS. But at least we can add `minimum: 1,` to limit it.